pkgs/top-level/python-packages.nix

```diff
@@ -6837,6 +6837,9 @@ in {
 
   pony = callPackage ../development/python-modules/pony { };
 
+  xandikos = disabledIf (!isPy3k) (toPythonModule (pkgs.xandikos.override {
+    python3Packages = self;
+  }));
```

Comment:
feel like moving xandikos into python-modules and then doing toPythonApplication for the all-packages export would be more appropriate